packages/medusa/src/loaders/helpers/routing/index.ts

```diff
@@ -601,6 +606,21 @@ export class RoutesLoader {
         )
       }
 
+      if (descriptor.config.shouldAppendAuthCors) {
```

Comment:
Can you explain the logic behind having a different CORS setting for auth? Is this for the cases where the auth module can be hosted on a separate domain?

Reply:
This is to accept requests from whitelisted 3rd party redirects coming in from google and the lot.